apps/consulting/pages/_app.tsx

```diff
@@ -1,13 +1,26 @@
 import { AppProps } from 'next/app';
+import Script from 'next/script';
 
 import { Meta } from '@quansight/shared/ui-components';
+
 import './styles.css';
 
 function CustomApp({ Component, pageProps }: AppProps): JSX.Element {
+  const consultingDomain = 'quansight.com';
   return (
     <>
       <Meta />
       <Component {...pageProps} />
+
+      {typeof window !== 'undefined' &&
+        window.location.hostname === consultingDomain && (
```

Comment:
The idea here is that we don't want to send events to Plausible analytics when we're testing things on the vercel.app domains. Plausible excludes localhost by default but doesn't know how to exclude other domains.